docs/docs/installation/pypi.mdx

```diff
@@ -22,13 +22,13 @@ level dependencies.
 
 **Debian and Ubuntu**
 
-The following command will ensure that the required dependencies are installed:
+In Ubuntu **before 20.04** the following command will ensure that the required dependencies are installed:
 
 ```bash
 sudo apt-get install build-essential libssl-dev libffi-dev python-dev python-pip libsasl2-dev libldap2-dev default-libmysqlclient-dev
 ```
 
-In Ubuntu 20.04 the following command will ensure that the required dependencies are installed:
+In Ubuntu **20.04 and 22.04** the following command will ensure that the required dependencies are installed:
```

Comment:
What happens if it's a Ubuntu version that is not in the range? Maybe add this additional context.

Reply:
Sadly, apparently superset can't be installed yet on Ubuntu 24.04. TLDR: the included `numpy` is too old.

Longer story:

- Ubuntu 24.04 has `python3.12`.
- `numpy` below 1.26.4 is incompatible with `python3.12`.
- Superset requires `numpy==1.23.5`.

Apparently, there was an issue about this earlier, but it was closed without resolution: https://github.com/apache/superset/issues/27585